hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/metadata/DatanodeStoreSchemaThreeImpl.java

```diff
@@ -40,7 +40,8 @@
  * - All keys have containerID as prefix.
  * - The table 3 has String as key instead of Long since we want to use prefix.
  */
-public class DatanodeStoreSchemaThreeImpl extends AbstractDatanodeStore {
+public class DatanodeStoreSchemaThreeImpl extends AbstractDatanodeStore
+    implements DeleteTransactionStore<String> {
```

Comment:
"Rename dropAllWithPrefix  to removeContainerFromDB". Change is missed from last patch.

Reply:
Ah, I think I just renamed the helper function in `BlockUtils` but not the one in `DatanodeStoreSchemaThreeImpl`, let's rename it to removeContainerFromDB.